### ISSUE TITLE
Make it easier to install into a project

### ISSUE THREAD

**Issue by isaacabraham** [ionide/ionide-vscode-paket#72]
There's a nice feature to install a package and add to a project at the same time. However, you need to actually open up the proj file in the editor first to "select" it. This isn't especially obvious to do and a little awkward e.g. you're using Ionide's project view but need to go back to file view etc.

I suggest having the palette actually prompt you to select the project you wish to add the package to as a drop-down, if this is even possible.

**Comment by Krzysztof-Cieslak** [comment]
Yes, I agree. Some of the UX of Paket plugin comes from the time when we’ve been just starting with VSCode plugins. It’s hard to me to say now if that particular design was caused by more limited API back in early days, or by just my “views” on something. But in 2019 we definitely can do better :-)
